src/Mod/Sketcher/App/planegcs/Constraints.cpp

```diff
@@ -232,9 +212,10 @@ double ConstraintWeightedLinearCombination::grad(double* param)
 ConstraintCenterOfGravity::ConstraintCenterOfGravity(const std::vector<double*>& givenpvec,
                                                      const std::vector<double>& givenweights)
     : weights(givenweights)
+    , numpoints(pvec.size() - 1)
```

Comment:
Doesn't this need to be `givenpvec` to actually match the original, since `pvec` won't have been assigned yet?

Reply:
Yes